app/subscriber/src/features/my-reports/edit/content/stories/ContentForm.tsx

```diff
@@ -152,6 +253,26 @@ export const ContentForm: React.FC<IContentFormProps> = ({
             onContentChange?.(values);
           }}
         />
+        <Formik
```

Comment:
Change the `FormikSentiment` to work in a way that is intuitive.  Using a sub-form just to get around this issue is confusing and difficult to maintain.

Reply:
Implemented another component.